mod-source-record-storage-server/src/main/java/org/folio/services/handlers/actions/AbstractUpdateModifyEventHandler.java

```diff
@@ -124,7 +124,7 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload p
           return recordService.saveRecord(changedRecord, payload.getTenant());
         })
         .onSuccess(savedRecord -> {
-          payload.setEventType(getNextEventType());
+          payload.setEventType(getUpdateEventType());
```

Comment:
If the success handling is the only thing that is different from the abstract method implementation, maybe it is worth extracting it in a separate method and overriding it for each handler. It should reduce the code duplication